pleskdistup/actions/extensions.py

```diff
@@ -133,3 +133,33 @@ def _revert_action(self) -> action.ActionResult:
 
     def estimate_post_time(self) -> int:
         return 60 * len(self._get_ruby_domains())
+
+
+class UninstallTuxcareEls(action.ActiveAction):
+    """ TuxCare ELS extension is installed on EoL OSes and may enable repositories incompatible with
+        the target OS version. Uninstalling also removes the repositories.
+    """
+
+    def __init__(self):
+        self.name = "uninstall tuxcare-els"
+        self.ext_name = "tuxcare-els"
```

Comment:
Please declare `ext_name`'s type on class level for type checking.

Reply:
Type checking is not set up properly anyway, so this has no bearing on the checks result or code correctness.
Also mypy is relatively smart and explicit type annotations for class members are often excessive (as long as the constructor is treated as typed, which is not the case here and in majority of other existing code here).
I'll fix it, but I see no current value in this.